packages/protocol/contracts/identity/test/MockRandom.sol

```diff
@@ -1,13 +1,20 @@
 pragma solidity ^0.5.3;
 
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "../Random.sol";
 
+contract MockRandom is Random {
+  uint256 public randomnessBlockRetentionWindow = 256;
```

Comment:
Where is this used?

Reply:
Somehow I thought I ran into an issue and so I just copied the implementation in here, but is fine, so I removed it!